REUSE.toml

```diff
@@ -0,0 +1,90 @@
+# SPDX-License-Identifier: curl
+# SPDX-FileCopyrightText: Daniel Stenberg, <daniel@haxx.se>, et al.
+
+version = 1
+SPDX-PackageName = "curl"
+SPDX-PackageSupplier = "Daniel Stenberg <daniel@haxx.se>"
+SPDX-PackageDownloadLocation = "https://curl.se/"
+
+# /
+[[annotations]]
+path = [
+  ".mailmap",
+  "CHANGES",
+  "GIT-INFO.md",
+  "libcurl.def",
+  "README",
+  "RELEASE-NOTES",
+  "renovate.json",
+]
+SPDX-FileCopyrightText = "Daniel Stenberg, <daniel@haxx.se>, et al."
+SPDX-License-Identifier = "curl"
+
+# docs/
+[[annotations]]
+path = [
+  "docs/FAQ",
+  "docs/INSTALL",
+  "docs/KNOWN_BUGS",
+  "docs/libcurl/symbols-in-versions",
+  "docs/MAIL-ETIQUETTE",
+  "docs/options-in-versions",
+  "docs/THANKS",
+  "docs/TODO",
+]
+SPDX-FileCopyrightText = "Daniel Stenberg, <daniel@haxx.se>, et al."
+SPDX-License-Identifier = "curl"
+
+# lib/
+[[annotations]]
+path = "lib/libcurl.vers.in"
+SPDX-FileCopyrightText = "Daniel Stenberg, <daniel@haxx.se>, et al."
+SPDX-License-Identifier = "curl"
+
+# packages/
+[[annotations]]
+path = [
+  "packages/OS400/README.OS400",
+  "packages/vms/build_vms.com",
+  "packages/vms/curl_release_note_start.txt",
+  "packages/vms/curlmsg.sdl",
+  "packages/vms/macro32_exactcase.patch",
+  "packages/vms/readme",
+]
+SPDX-FileCopyrightText = "Daniel Stenberg, <daniel@haxx.se>, et al."
+SPDX-License-Identifier = "curl"
+
+# plan9/
+[[annotations]]
+path = "plan9/README"
+SPDX-FileCopyrightText = "Daniel Stenberg, <daniel@haxx.se>, et al."
+SPDX-License-Identifier = "curl"
+
+# projects/
+[[annotations]]
+path = ["projects/Windows/**", "projects/wolfssl_override.props"]
+SPDX-FileCopyrightText = "Daniel Stenberg, <daniel@haxx.se>, et al."
+SPDX-License-Identifier = "curl"
+
+# tests/
+[[annotations]]
+path = [
+  "tests/certs/**",
+  "tests/data/test**",
+  "tests/stunnel.pem",
+  "tests/valgrind.supp",
+]
+SPDX-FileCopyrightText = "Daniel Stenberg, <daniel@haxx.se>, et al."
+SPDX-License-Identifier = "curl"
+
+# checksrc control files
+[[annotations]]
+path = [
+  "docs/examples/.checksrc",
+  "lib/.checksrc",
+  "src/.checksrc",
+  "tests/libtest/.checksrc",
+
+]
```

Comment:
I think this could be simplified with all files in a single list as they have the same copyright and same license.

Reply:
That's possible of course. Why didn't we/you do this with the old dep5 file? I figured it there to somehow get a better overview of these files.

Feel free to make a decision, I don't have a preference.

Reply:
I have thought about making it so in the old file as well but never got around to it but since it is now getting an overhaul anyway I think it is worth doing that. It simplifies the file a lot and removes repetition outside of the file lists.

Reply:
OK, did that in my recent commits!